tests/ui/test_home.py

```diff
@@ -1,70 +1,72 @@
 import pytest
 
+
```

Comment:
Imho that's one newline too much. One newline only should be used to separate import groups. Feel free to run https://pypi.python.org/pypi/isort over your code, we do have a proper config for it, we just don't enforce it yet (WIP)